lib/full_url_for.ts

```diff
@@ -25,7 +25,7 @@ function fullUrlForHelper(path = '/') {
     path = prettyUrls(path, prettyUrlsOptions);
 
     return path;
-  });
+  }) as string;
```

Comment:
The type cast here is unnecessary. Just improve `cache.apply`'s type declaration instead.